Source/WebCore/css/parser/CSSPropertyParserWorkerSafe.cpp

```diff
@@ -223,6 +223,8 @@ static RefPtr<CSSFontFaceSrcResourceValue> consumeFontFaceSrcURI(CSSParserTokenR
             return nullptr;
         format = arg.value().toString();
     }
+    if (!range.atEnd())
+        return nullptr;
```

Comment:
Is this tested?

Reply:
yes, I believe we should invalidate a component (but not the whole descriptor) if, at the end of parsing, the range is not at the end, meaning we haven't consumed all of it.  For example:
` { src: 'url("foo.ttf") dummy(xyzzy), url("bar.html")', valid: true },`
For this descriptor, the first component should be invalidated. For that, after consuming `url()`, we will just consume the next non-blank token if it's a `format()` (and in the future `tech()`). At the end of it, if the whole component was not consumed, this means that some garbage data was present and the component should be invalid.

Oh, while I wrote that I realized you are probably asking if we actually have a test for it, instead of if this behavior is intended. The fragment above comes from wpt font-face-src-format.html, which should be valid because of the second component and not the first. So we probably should add a subtest with the first component only, and valid:false. I'll create a separated PR and export it to WPT. We do have an internal fast test for it though at fast/css/font-face-src-parsing.html:

```
@font-face {
    src: url(foo.ttf) "invalid";
}
```